moped-editor/src/views/projects/projectView/ProjectSummaryMapFallback.js

```diff
@@ -21,6 +21,12 @@ const useStyles = makeStyles(theme => ({
   paragraphGroup: {
     margin: "1rem 0",
   },
+  mapPlaceholder: {
+    color: "#848484",
```

Comment:
this could be `theme.palette.text.secondary` so if we update that color later, we dont have to go around hunting for everywhere with secondary text